test/test_routes.py

```diff
@@ -1548,3 +1550,58 @@ def test_bash_api_multiple_inputs_outputs():
         assert response.status_code == 200
         assert "event: complete\ndata:" in response.text
         assert json.dumps([123, "abc"]) in response.text
+
+
+def test_attacker_cannot_change_root_in_config(
```

Comment:
Great test! But it takes about a minute to run. I wonder if it can be optimized to run faster or only run on release prs

Reply:
Really, are you sure? I was worried about this, so I ran this a few times, and it takes about 5 seconds on my machine